inyoka_theme_default/templates/portal/privmsg/index.html

```diff
@@ -0,0 +1,232 @@
+{#
+  portal/privmsg/index.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Display a list of private messages.
+
+  :copyright: (c) 2013-2015 by the Inyoka Team, see AUTHORS for more details.
+  :license: BSD, see LICENSE for more details.
+#}
+
+{% extends 'portal/privmsg/base.html' %}
+
+{% block breadcrumb %}
+  {{ super() }}
+  {{ macros.breadcrumb_item(_(folder.name)) }}
+{% endblock %}
+
+{% block content %}
+  {% if message %}
+    <div class="message">
+      <div class="head">
+        <div class="row">
+          <div class="col-sm-9">
+            <div class="row">
+              <div class="col-sm-3">
+                <strong>{% trans %}From:{% endtrans %}</strong>
+              </div>
+              <div class="col-sm-9">
+                <a{% if not message.author.is_active %} class="user_inactive"{% endif %} href="{{ message.author|url }}">
```

Comment:
nope.
I think we can do something like "Background:grey; opacity:0.7" vor inactive users?